### PR TITLE
CMS folder topic tune up GH 547 - content elements

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is part 2 of the clean up efforts for #547 and addresses the content menu and elements topics in the CMS folder. Upon walkthrough of the procedures, a few screens were also added or updated as needed. There was some reorganization of topics for better structure and consistency, as well as the following items:

- Captions using sentence case
- Headings using sentence case and not infinitives
- Consistency in numbered step structure and style
- Edits for clarity

Related PR for #547: https://github.com/magento/merchdocs/pull/746

Note: This will be applied first to the 2.3.x production docs and then forward ported.

**Internal staging build at: 542/user-guide/v2.3/**

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/v2.3/cms/adobe-stock-manage.html
https://docs.magento.com/user-guide/v2.3/cms/adobe-stock.html
https://docs.magento.com/user-guide/v2.3/cms/block-add.html
https://docs.magento.com/user-guide/v2.3/cms/block-position.html
https://docs.magento.com/user-guide/v2.3/cms/blocks.html
https://docs.magento.com/user-guide/v2.3/cms/content-elements.html
https://docs.magento.com/user-guide/v2.3/cms/content-menu.html
https://docs.magento.com/user-guide/v2.3/cms/dynamic-blocks-price-rules.html
https://docs.magento.com/user-guide/v2.3/cms/dynamic-blocks-rotate.html
https://docs.magento.com/user-guide/v2.3/cms/dynamic-blocks.html
https://docs.magento.com/user-guide/v2.3/cms/editor-configure.html
https://docs.magento.com/user-guide/v2.3/cms/editor-insert-image.html
https://docs.magento.com/user-guide/v2.3/cms/editor-insert-link.html
https://docs.magento.com/user-guide/v2.3/cms/editor-insert-variable.html
https://docs.magento.com/user-guide/v2.3/cms/editor-widget.html
https://docs.magento.com/user-guide/v2.3/cms/editor.html
https://docs.magento.com/user-guide/v2.3/cms/lightboxes-and-sliders.html
https://docs.magento.com/user-guide/v2.3/cms/media-storage.html
https://docs.magento.com/user-guide/v2.3/cms/page-actions.html
https://docs.magento.com/user-guide/v2.3/cms/page-add.html
https://docs.magento.com/user-guide/v2.3/cms/page-duplicate.html
https://docs.magento.com/user-guide/v2.3/cms/page-hierarchy-configure.html
https://docs.magento.com/user-guide/v2.3/cms/page-hierarchy-node-add.html
https://docs.magento.com/user-guide/v2.3/cms/page-hierarchy.html
https://docs.magento.com/user-guide/v2.3/cms/page-home-new.html
https://docs.magento.com/user-guide/v2.3/cms/page-scheduled-changes.html
https://docs.magento.com/user-guide/v2.3/cms/page-translate.html
https://docs.magento.com/user-guide/v2.3/cms/pages-core.html
https://docs.magento.com/user-guide/v2.3/cms/pages-default.html
https://docs.magento.com/user-guide/v2.3/cms/pages-grid-columns.html
https://docs.magento.com/user-guide/v2.3/cms/pages-search.html
https://docs.magento.com/user-guide/v2.3/cms/pages-workspace.html
https://docs.magento.com/user-guide/v2.3/cms/pages.html
https://docs.magento.com/user-guide/v2.3/cms/widget-create.html
https://docs.magento.com/user-guide/v2.3/cms/widget-event-carousel.html
https://docs.magento.com/user-guide/v2.3/cms/widget-new-products-list.html
https://docs.magento.com/user-guide/v2.3/cms/widget-orders-returns.html
https://docs.magento.com/user-guide/v2.3/cms/widget-static-block.html
https://docs.magento.com/user-guide/v2.3/cms/widget-types.html
https://docs.magento.com/user-guide/v2.3/cms/widgets.html

### CODE REVIEW

File: src/cms/page-add.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ _Pages Grid_{:.ee-only}
 
 1. Complete the content, and format the text as needed.
 
-   To learn more, see [Workspace]({% link cms/page-builder-workspace.md %}). If PageBuilder is not enabled, the editor opens in WYSIWYG mode with the toolbar at the top.
+   For more information, see [Workspace]({% link cms/page-builder-workspace.md %}). If PageBuilder is not enabled, the editor opens in WYSIWYG mode with the toolbar at the top.
```
Comment (hickskenh) [inline src/cms/page-add.md:64]:
```suggestion
   For more information, see [Workspace]({% link cms/page-builder-workspace.md %}). If _Page Builder_ is not enabled, the editor opens in WYSIWYG mode with the toolbar at the top.
```

File: src/cms/page-add.md [hunk 2]
```diff
@@ -128,11 +128,11 @@ _Pages Grid_{:.ee-only}
    - {: .ee-only}`Category -- Full Width` (Requires Page Builder)
    - {: .ee-only}`Product -- Full Width` (Requires Page Builder)
 
-1. To apply a **Custom Layout Update**, choose the name of the file from the list. To learn more, see [Layout Updates]({% link design/layout-updates.md %}).
+1. To apply a **Custom Layout Update**, choose the name of the file from the list. For more information, see [Layout Updates]({% link design/layout-updates.md %}).
```
Comment (hickskenh) [inline src/cms/page-add.md:131]:
```suggestion
1. To apply a **Custom Layout Update**, choose the name of the file from the list.
```

File: src/cms/page-duplicate.md [hunk 3]
```diff
@@ -5,46 +5,44 @@ title: Duplicating a Page
...
 
-1. Open the duplicate page in **Edit** mode. Then, do the following:
+1. Open the duplicate page in _Edit_ mode and do the following:
 
    - If you want to publish the page at this time, set **Enable Page** to `Yes`.
 
    - Update the **Page Title**, as needed.
 
-   - Expand the **Search Engine Optimization** section. Then, enter the unique **URL Key** that you want to use for the duplicate page.
+   - Expand the **Search Engine Optimization** section and enter the unique **URL Key** that you want to use for the duplicate page.
```
Comment (hickskenh) [inline src/cms/page-duplicate.md:33]:
```suggestion
   - Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Search Engine Optimization** section and enter the unique **URL Key** that you want to use for the duplicate page.
```

File: src/cms/pages-default.md [hunk 4]
```diff
@@ -2,36 +2,36 @@
 title: Default Pages
 ---
 
-The Default Pages configuration determines the landing page that is associated with the [base URL]({% link stores/store-urls.md %}), and the corresponding home page. It also determines which page appears when a “Page Not Found” error occurs, and if a [breadcrumb trail]({% link catalog/navigation-breadcrumb-trail.md %}) appears at the top of each page.
+The Default Pages configuration determines the landing page that is associated with the [base URL]({% link stores/store-urls.md %}) and the corresponding home page. It also determines which page appears when a _Page Not Found_ error occurs, and if a [breadcrumb trail]({% link catalog/navigation-breadcrumb-trail.md %}) appears at the top of each page.
```
Comment (hickskenh) [inline src/cms/pages-default.md:5]:
```suggestion
The _Default Pages_ configuration determines the landing page that is associated with the [base URL]({% link stores/store-urls.md %}) and the corresponding home page. It also determines which page appears when a _Page Not Found_ error occurs, and if a [breadcrumb trail]({% link catalog/navigation-breadcrumb-trail.md %}) appears at the top of each page.
```

File: src/cms/widget-create.md [hunk 5]
```diff
@@ -2,79 +2,90 @@
...
 
-1. In the panel on the left, choose **Widget Options**.
+1. In the left panel, choose **Widget Options**.
 
 1. Click <span class="btn">Select Block</span>.
 
-1. Enter a **Title** to appear above the list. For example, `Featured Products`.
+1. Enter a **Title** to appear above the list.
+
+   For example, `Featured Products`.
```
Comment (hickskenh) [inline src/cms/widget-create.md:68]: this isn't a sentence.    suggest moving to first line with , such as or e.g.

File: src/cms/widget-event-carousel.md [hunk 6]
```diff
@@ -4,84 +4,93 @@ title: Catalog Events Carousel
...
+   _Widget settings for an event carousel_
 
-1. In the **Storefront Properties** section, complete the following fields:
+1. In the _Storefront Properties_ section, do the following:
 
-    |---|---|
-    | Widget Title | Enter a descriptive title for your widget. This title is visible only from the Admin. |
-    | Assign to Store Views | Select the store views where the widget will be visible. You can select a specific store view, or "All Store Views". |
-    | Sort Order | (Optional) Enter a number to determine the order this items appears with others in the same part of the page. (0 = first, 1 = second, 3= third, and so on.) |
+   - For **Widget Title**, enter a descriptive title for the widget. (This title is visible only from the _Admin_.)
```
Comment (hickskenh) [inline src/cms/widget-event-carousel.md:38]:
```suggestion
   - For **Widget Title**, enter a descriptive title for the widget. 
   
      This title is visible only from the _Admin_.
```

File: src/cms/widget-new-products-list.md [hunk 7]
```diff
@@ -2,108 +2,116 @@
...
+1. Set **Number of Products to Display** to the number of new products that you want to include in the list.
 
-   - To provide navigation control for lists with multiple pages, set **Display Page Control** to “Yes”. Then, in the **Number of Products per Page** field, enter the number of products you want to appear on each page.
+   The default setting is `10`.
 
-   - Set **Number of Products to Display** to the number of new products that you want to include in the list. The default setting is 10.
+1. For **Cache Lifetime (Seconds)**, choose how often you want to refresh the list of new products.
 
-   - In the **Cache Lifetime (Seconds)** field, choose how often you want to refresh the list of new products. By default, the cache is set to 86400 seconds, or 24 hours.
+   By default, the cache is set to 86400 seconds, or 24 hours.
```
Comment (hickskenh) [inline src/cms/widget-new-products-list.md:104]:
```suggestion
   By default, the cache is set to 86,400 seconds (24 hours).
```

File: src/cms/widget-static-block.md [hunk 8]
```diff
@@ -1,62 +1,67 @@
...
-    - Click <span class="btn">Continue</span>.
+   ![]({% link images/images/widget-settings.png %}){: .zoom}
+   _Widget settings_
 
-    ![]({% link images/images/widget-settings.png %}){: .zoom}
-    _Settings_
+1. In the _Storefront Properties_ section, do the following:
 
-1. In the Storefront Properties section, do the following:
+   - For **Widget Title**, enter a descriptive title for the widget. This title is visible only from the _Admin_.
```
Comment (hickskenh) [inline src/cms/widget-static-block.md:25]:
```suggestion
   - For **Widget Title**, enter a descriptive title for the widget. 
   
         This title is visible only from the _Admin_.
```

**Review by hickskenh — APPROVED** [review]
approved after your review.

File: src/cms/content-menu.md [hunk 9]
```diff
@@ -1,45 +1,43 @@
...
+The term _content marketing_ refers to the art of promoting your products or services by providing valuable information to your customers at no charge. The quality of your content helps distinguish your store from others, increases your visibility to search engines, and provides support to your customers. This soft-sell approach is often more effective than advertising, builds credibility and trust, and can turn your store into a destination. Content is still king.
 
 Your content should reflect the branding of your store, and be delivered with your distinctive visual presentation and voice to convey your message. Use pictures to tell a story. Educate, inspire, and entertain. Keep in mind that with quality content, sometimes less is more.
 
-![]({% link images/images-ee/admin-menu-content-ee.png %}){: .zoom}
-_Content Menu_{:.ee-only}
+![The Content menu displayed in the Magento Admin]({% link images/images-ee/admin-menu-content-ee.png %}){: .zoom}
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/cms/content-menu.md:11]: This is the alt text so formatting is moot

File: src/cms/adobe-stock.md [hunk 10]
```diff
@@ -7,52 +7,60 @@ Merchants can integrate [Adobe Stock][adobe-stock] with Magento giving them acce
 ![Adobe Stock Search Results]({% link images/images/adobe-stock/search-grid.png %}){: .zoom}
 
 The Adobe Stock service provides businesses with access to millions of high-quality, curated, royalty-free photos, vectors, illustrations, videos, templates, and 3D assets for all their creative projects. Magento users are able to quickly find, preview, and license Adobe Stock assets. Users can also save them to the
-[Media Storage][media-storage], all without leaving the Magento workspace.
+[Media Storage][media-storage], all without leaving the Magento Admin workspace.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/cms/adobe-stock.md:10]: double-checked, nope

File: src/cms/adobe-stock.md [hunk 11]
```diff
@@ -7,52 +7,60 @@ Merchants can integrate [Adobe Stock][adobe-stock] with Magento giving them acce
...
+Licensing Adobe Stock images requires:
+
+- An [Adobe account][adobe-signin]
+- A paid [Adobe Stock][adobe-stock] plan associated with the account
 
 ## Integrate Magento and Adobe Stock
 
-Configuring the Adobe Stock Integration for Magento is a two-step process: [create an adobe.io integration][create-integration] to generate an API Key, and
-then [configure the Adobe Stock Integration in your Magento Admin][configure].
+Configuring the Adobe Stock integration for Magento is a two-step process: [create an adobe.io integration][create-integration] to generate an API Key, and then [configure the Adobe Stock integration in your Magento Admin][configure].
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/cms/adobe-stock.md:26]: I thought about this but I did not like how it conveys with one of the procedures afterward using the same formatted numbering. I think I will leave for now but think on other alternative.

File: src/cms/dynamic-blocks-price-rules.md [hunk 12]
```diff
@@ -40,22 +40,22 @@ Any [dynamic block]({% link cms/dynamic-blocks.md %}) that you create can be ass
...
-    - To associate a shopping cart promotion, click <span class="btn">Add Cart Price Rules</span>.
+   - To associate a shopping cart promotion, click <span class="btn">Add Cart Price Rules</span>.
 
-    - To associate a product-related promotion, click <span class="btn">Add Catalog Price Rules</span>.
+   - To associate a product-related promotion, click <span class="btn">Add Catalog Price Rules</span>.
 
-        ![]({% link images/images-ee/page-builder-dynamic-block-related-promotions.png %}){: .zoom}
-        _Related Promotions_
+      ![]({% link images/images-ee/page-builder-dynamic-block-related-promotions.png %}){: .zoom}
+      _Related promotions for a dynamic block_
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/cms/dynamic-blocks-price-rules.md:52]: double-checked, nope

File: src/cms/dynamic-blocks-rotate.md [hunk 13]
```diff
@@ -8,98 +8,99 @@ redirect_from:
...
+
+   This setting determines the type of dynamic blocks are included in the rotator.
 
-   - For the **Dynamic Blocks to Display**, accept `Specified Dynamic Blocks`.
+   Specified Dynamic Blocks|Includes only specific dynamic blocks.|
+   Cart Price Rule Related|Includes only dynamic blocks that are associated with a cart price rule.|
+   Catalog Price Rule Related|Includes only dynamic blocks that are associated a with catalog price rule.|
 
-   - To **Restrict the Dynamic Block Types** that can be used with the widget, select `Content Area`.
+|Restrict by Dynamic Block Types|Limits the banner to a specific part of the page layout. Options: <br/>
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/cms/dynamic-blocks-rotate.md:82]: This was a boo boo. Deleting.

File: src/cms/widget-create.md (hunk 5 above)
Comment (leslietilling) [reply src/cms/widget-create.md:68]: good call

**Comment by leslietilling** [comment]
running tests

**Pull request merged**